Add django, isa-l and libaom files (#138)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/django.md

---

> name: django

Author: ```suggestion
name: Django
```

> category: Web

Author: There is no category "**Web**" in the category sheet. 
Django is a python framework for building apps. 
I think "**Languages and Frameworks**" is the correct category for Django.

description: Django is a high-level Python web framework that encourages rapid development and clean, pragmatic design.

> download_url: https://github.com/django/django/tags?after=1.5a1

Author: ```suggestion
download_url: https://github.com/django/django/tags
```

works_on_arm: true
supported_minimum_version:
    version_number: 1.2.6
    release_date: 08/05/2012

optional_info:
    homepage_url: https://www.djangoproject.com/
    support_caveats:

>     alternative_options: https://github.com/django/django/blob/1.2.6/INSTALL

Author: remove this

    getting_started_resources:

>         arm_content: 

Author: ```suggestion
        arm_content: https://learn.arm.com/learning-paths/servers-and-cloud-computing/django/
```

        partner_content: 

>         official_docs: https://docs.djangoproject.com/en/5.0/releases/1.2.6/

Author: ```suggestion
        official_docs: https://docs.djangoproject.com/
```

    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info:

Author: add other info like no release notes are available. Please refer older raised PRs.

>     other_info: The first version of Django released in 2011 is 1.2.6. This version has been tested successfully on the Neoverse N1 Linux/ARM64. We picked this version as the minimum for Django, since ARM64 was introduced in the market in the same year.

Author: ```suggestion
    other_info: There are no release notes present for arm64 release. 1.2.6 version was built and tested successfully using released tar file.
```
